### DISCUSSION TITLE
Changes inside editorconfig.lua are ignored in my build

### DISCUSSION THREAD

**Discussion by marcg03** [neovim/neovim#28635] [Q&A] [answered]
I recently opened issue #28626 and I want to tackle it.

I searched for files related to **editorconfig** in the project after I cloned it and found `runtime/lua/editorconfig.lua`.
This file has instructions about adding support for editorconfig-properties so I think I found the right place to make the changes.

### The problem is: changes to  `runtime/lua/editorconfig.lua` seem to not affect the build.

I added a `print('Hello world')` to the first line of the already existing method `indent_size` to get some output. Afterwards I ran `make clean && make CMAKE_BUILD_TYPE=Debug`.

After the executable built I created a simple **.editorconfig** file that looks like this:
```
[*]
indent_size = 2
```
Then I ran `./nvim --clean .editorconfig`.
The `shiftwidth` option gets set correctly so it's clear that `nvim` successfully parsed the **.editorconfig** file but my `Hello world` is nowhere to be found (I even ran `:messages` to check if it's there).

### What I double checked
* I checked that I'm running the executable I just built.
* I switched `indent_size` from `2` to `4` inside **.editorconfig** to see that `nvim` is reacting. (It did react)
* I switched from `print('Hello world')` to `error('Hello world')` in case `print` doesn't work for some reason.

**Comment by zeertzjq** [comment, marked as answer]
What is `./nvim`? The built Nvim executable is `./build/bin/nvim`.

Please see [BUILD.md](https://github.com/neovim/neovim/blob/master/BUILD.md):
> - After building, you can run the `nvim` executable without installing it by running `VIMRUNTIME=runtime ./build/bin/nvim`.

**Reply by marcg03** [reply]
I `cd`-ed into ./build/bin. But I will run it as it says in BUILD.md and see if that was my problem.

**Reply by marcg03** [reply]
It was indeed the problem. Sorry for missing that in BUILD.md.
